server/tests/testUtils.ts

```diff
@@ -0,0 +1,13 @@
+import UserService from '../services/userService'
+
+export class MockUserService extends UserService {
+  constructor() {
+    super(undefined)
```

Comment:
does it need undefined here?

Reply:
Yeah, if I remove it TypeScript complains with `Expected 1 arguments, but got 0.`